src/artifacts/mod.rs

```diff
@@ -1467,7 +1467,9 @@ impl Source {
     pub fn read(file: impl AsRef<Path>) -> Result<Self, SolcIoError> {
         let file = file.as_ref();
         trace!(file=%file.display());
-        let content = fs::read_to_string(file).map_err(|err| SolcIoError::new(err, file))?;
+        let content = fs::read_to_string(file)
+            .map_err(|err| SolcIoError::new(err, file))?
+            .replace("\r\n", "\n");
```

Comment:
should we only do this on windows?

this is probably not a big perf hit, but if we can limit this to windows only, it would be ideal